action_test.py

```diff
@@ -0,0 +1,75 @@
+import hashlib
+from unittest import TestCase
+
+from action import ContainerConfig, ContainerMount, prepare_dockerrun_file
+
+
+class MountsTest(TestCase):
+    HOST_VOLUME_PATH = "/var/my-sample-application-test/data"
+    HOST_VOLUME_NAME = hashlib.sha1(HOST_VOLUME_PATH.encode()).hexdigest()
+    APP_NAME = "sample-application"
+    IMAGE_NAME = "YourAwsAccountId.dkr.ecr.YourAwsRegion.amazonaws.com/your-ecr-repository"
+
+    def _create_container_config(self, container_path: str):
+        return ContainerConfig(
+            dockerfile="docker/Dockerfile",
+            name=self.APP_NAME,
+            image_base_name=self.IMAGE_NAME,
+            ports=(),
+            mounts=(
+                ContainerMount(
+                    host_path=self.HOST_VOLUME_PATH,
+                    container_path=container_path,
+                    read_only=False,
+                ),
+            ),
+        )
+
+    def _create_dockerrun_config(self, app_version: str, *mounts: str):
+        return {
+            "AWSEBDockerrunVersion": 2,
+            "volumes": [
+                {
+                    "name": self.HOST_VOLUME_NAME,
+                    "host": {
+                        "sourcePath": self.HOST_VOLUME_PATH,
+                    },
+                },
+            ],
+            "containerDefinitions": [
+                {
+                    "name": self.APP_NAME,
+                    "image": f"{self.IMAGE_NAME}:{app_version}",
+                    "environment": [],
+                    "essential": True,
+                    "links": (),
```

Comment:
```suggestion
                    "links": [],
```

Reply:
Das funktionier so leider nicht, da das von der Config-Dataclass durchgereicht wird, wo das Feld als Tupel deklariert ist. Wenn ich das ändere, schlagen die Tests fehl.

Reply:
Aber in dem `Dockerrun.aws.json` ist das doch nachher eine Liste und kein Tuple.